@xen-orchestra/web-core/lib/components/input/InputTextArea.vue

```diff
@@ -0,0 +1,91 @@
+<!-- v1 -->
+<template>
+  <div class="vts-input-text-area">
+    <VtsLabel v-if="$slots.default" :accent="labelAccent" :required :icon :href><slot /></VtsLabel>
+    <textarea v-model="model" :class="accent" :disabled class="text-area" v-bind="$attrs" />
+    <VtsInfo v-if="$slots.info" :accent><slot name="info" /></VtsInfo>
+  </div>
+</template>
+
+<script lang="ts" setup>
+import VtsInfo from '@core/components/info/VtsInfo.vue'
+import VtsLabel from '@core/components/input/VtsLabel.vue'
+import type { IconDefinition } from '@fortawesome/fontawesome-common-types'
+import { computed } from 'vue'
+
+defineOptions({
+  inheritAttrs: false,
+})
+
+const props = defineProps<{
+  accent: 'brand' | 'warning' | 'danger'
+  disabled?: boolean
+  href?: string
+  icon?: IconDefinition
+  required?: boolean
+}>()
+
+const model = defineModel<string>({ required: true })
+
+defineSlots<{
+  default(): any
+  label(): any
```

Comment:
BTW, there is a `label` property.
I think this ties into our conversation about "when" to use slots and when to use props.
IMO, in that case, slot should be used instead of a prop